ui/packages/catalog-ui-search/src/main/webapp/react-component/inputs/enum-input/enum-input.js

```diff
@@ -53,23 +67,49 @@ const EnumInput = ({
           onChange={setInput}
         />
       </TextWrapper>
-      <Menu value={value} onChange={onChange}>
+      <Menu value={value} onChange={onChange} class="fa">
         {allowCustom &&
           displayInput && (
             <EnumMenuItem value={input}>{input} (custom)</EnumMenuItem>
           )}
         {filteredSuggestions.map(suggestion => {
           return (
-            <EnumMenuItem key={suggestion.value} value={suggestion.value}>
+            <EnumMenuItem
+              title={
+                isAttributeDisabled(settingsModel,suggestion.value)
+                  ? 'Attribute is unsupported by the content store(s) selected'
+                  : ''
+              }
+              key={suggestion.value}
+              value={suggestion.value}
+              disabled={isAttributeDisabled(settingsModel,suggestion.value)}
+            >
               {suggestion.label}
             </EnumMenuItem>
           )
         })}
       </Menu>
     </Dropdown>
   )
+  
+  return (
+    <div>
+      {isAttributeDisabled(settingsModel,selected.value) ? (
```

Comment:
```suggestion
      {isAttributeDisabled(settingsModel, selected.value) ? (
```
brought to you by `/projects/ddf/ui>> yarn format -w`